src/components/TextInput/TextInput.scss

```diff
@@ -129,10 +172,12 @@ $block: '.#{variables.$ns}text-input';
 
         &_xl {
             #{$block}__control {
-                height: 44px;
-                padding: 12px 13px;
-                font-size: 15px;
-                line-height: 20px;
+                @include block_size(xl);
+            }
+
+            #{$block}__label {
+                @include block_size(xl);
+                padding-right: 4px;
```

Comment:
Can be lifted up for deduplication

Reply:
it's done like this to override `padding` property in mixins

Reply:
ok